internal/config/baseline.go

```
@@ -0,0 +1,135 @@
+	}
+	data, err := json.MarshalIndent(b, "", "  ")
+	if err != nil {
+		return err
+	}
+	tmp, err := os.CreateTemp(filepath.Dir(path), ".scan-baselines-*.json")
+	if err != nil {
+		return err
+	}
+	tmpName := tmp.Name()
+	if _, err := tmp.Write(data); err != nil {
+		tmp.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `tmp.Close` is not checked

(errcheck)